### ISSUE TITLE
CSS 'transition all' affects overwrites

### ISSUE THREAD

**Issue by BahaMagi** [hc-oss/react-multi-select-component#473]
Some classes like `select-item` classes have transitions set as `transition: all .2s ease;`. This messes with custom overwrites. It would be nice to have more targeted css transitions to ensure that custom overwrites for example on hover effects dont get affected.

**Comment by harshzalavadiya** [comment]
@BahaMagi transition is also simple css, you should be easily be able to override or remove it by using below CSS rule

```css
.rmsc * {
  transition: initial; // or override
}
```

**Comment by BahaMagi** [comment]
Thank you for the quick response! I solved it in my case by just removing the transitions on the components I changed:
```
.select-item * {
    transition: none;
  }
```
Argument was rather that if I wanted to keep existing transitions but add new styling to for example the hover state the transitions will automatically take effect. 
If the transitions for the default rendering/styles were targeting the props they are supposed to affect it would be easier/more straight forwards to extend. But I guess it boils down to more of a coding style question than an issue.
